node/src/server_initializer.rs

```diff
@@ -695,20 +676,25 @@ pub mod tests {
         tlh.exists_log_containing("ERROR: PanicHandler: file.txt:24:42 - I'm just a string slice");
     }
 
+    pub fn make_pre_populated_mocked_directory_wrapper() -> DirsWrapperMock {
+        DirsWrapperMock::new()
+            .home_dir_result(Some(PathBuf::from("/home/alice")))
```

Comment:
If you run into problems with Windows here, try using PathBuf::join() instead of slashes.